jax/_src/compute_on.py

```diff
@@ -40,10 +40,12 @@ def current_compute_type() -> str | None:
   return config.compute_on_context_manager.value
 
 def _check_valid(c_type: str):
-  if c_type not in {'device_host', 'device', 'tpu_sparsecore'}:
+  if (c_type not in {'device_host', 'device', 'tpu_sparsecore'}
+      and not c_type.startswith("stream:")):
```

Comment:
I wonder if this should be named `gpu_stream:`. Because on TPU it doesn't make sense right?

WDYT?

Reply:
I think that's a good idea, because it doesn't make sense on CPU either.

Reply:
Cool! Is the xla bug fixed? If yes, enable the test and we can get this in :)